lib/manager/npm/post-update/lerna.ts

```diff
@@ -55,8 +55,10 @@ export async function generateLockFiles(
       logger.warn({ lernaClient }, 'Unknown lernaClient');
       return { error: false };
     }
+    let lernaCommand = `lerna bootstrap --no-ci --ignore-scripts -- ${cmdOptions}`;
     if (global.trustLevel === 'high' && config.ignoreScripts !== false) {
       cmdOptions = cmdOptions.replace('--ignore-scripts ', '');
+      lernaCommand = lernaCommand.replace('--ignore-scripts ', '');
```

Comment:
I think this should use the regex version of `replace` to replace all instead of just the first

> If pattern is a string, only the first occurrence will be replaced.
> \- [source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)

Reply:
Your latest changes makes this comment irrelevant